tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -11,32 +11,32 @@ weave_options:
 Principal component analysis (PCA) is a fundamental technique to analyse and visualise data.
 It is an unsupervised learning method mainly used for dimensionality reduction.
 
-For example, we have a data matrix $\mathbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k \ll D$ principal components which captures most of the information from the original matrix.
-The goal is to understand $\mathbf{X}$ through a lower dimensional subspace (e.g. two-dimensional subspace for visualisation convenience) spanned by the principal components.
+For example, we have a data matrix $\textbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k \ll D$ principal components which captures most of the information from the original matrix.
+The goal is to understand $\textbf{X}$ through a lower dimensional subspace (e.g. two-dimensional subspace for visualisation convenience) spanned by the principal components.
 
-In order to project the original data matrix into low dimensions, we need to find the principal directions where most of the variations of $\mathbf{X}$ lie in.
+In order to project the original data matrix into low dimensions, we need to find the principal directions where most of the variations of $\textbf{X}$ lie in.
 Traditionally, this is implemented via [singular value decomposition (SVD)](https://en.wikipedia.org/wiki/Singular_value_decomposition) which provides a robust and accurate computational framework for decomposing matrix into products of rotation-scaling-rotation matrices, particularly for large datasets(see an illustration [here](https://intoli.com/blog/pca-and-svd/)):
 
 $$
-\mathbf{X}_{N \times D} =  \mathbf{U}_{N \times r} \times \boldsymbol{\Sigma}_{r \times r}  \times  \mathbf{V}^T_{r \times D}
+\textbf{X}_{N \times D} =  \textbf{U}_{N \times r} \times \textbf{Σ}_{r \times r}  \times  \textbf{V}^T_{r \times D}
```

Comment:
I think `\boldsymbol` might be the only problematic part here. Also should use `\Sigma`.

Reply:
Agree, but with `Sigma` we need `boldsymbol`, while we can use `mathbf{Σ} to avoid `boldsymbol`?